eng/common/scripts/Verify-ChangeLogs.ps1

```diff
@@ -7,6 +7,28 @@ Set-StrictMode -Version 3
 
 . (Join-Path $PSScriptRoot common.ps1)
 
+function ShouldVerifyChangeLog ($ServiceDirectory, $PackageName) {
+    $jsonCiYmlPath = Join-Path $ServiceDirectory "ci.yml.json"
```

Comment:
I would suggest just looking for ci.yml and parsing it on the fly right here.

Reply:
Yeah chatted offline.